packages/babel-preset-default/package.json

```diff
@@ -38,9 +39,13 @@
 		"@wordpress/browserslist-config": "file:../browserslist-config",
 		"@wordpress/element": "file:../element",
 		"@wordpress/warning": "file:../warning",
-		"core-js": "^3.6.4"
+		"core-js": "^3.11.0",
+		"core-js-builder": "3.11.0"
```

Comment:
It's a dev dependency, so it probably can be moved to the top level `package.json`.

Reply:
Moved out.